xlators/mgmt/glusterd/src/glusterd-volume-set.c

```diff
@@ -2386,6 +2386,11 @@ struct volopt_map_entry glusterd_volopt_map[] = {
         .voltype = "storage/posix",
         .op_version = GD_OP_VERSION_3_13_0,
     },
+    {
+        .key = "storage.reserve-check-interval",
+        .voltype = "storage/posix",
+        .op_version = GD_OP_VERSION_4_0_0,
```

Comment:
This should be latest version where you can have this included. So GD_OP_VERSION_9_1